turbinia/client.py

```diff
@@ -1120,8 +1120,9 @@ def __init__(self, jobs_denylist=None, jobs_allowlist=None):
 
   def start(self):
     """Start Turbinia Server."""
-    log.info('Starting Prometheus endpoint.')
-    start_http_server(port=config.PROMETHEUS_PORT, addr=config.PROMETHEUS_ADDR)
+    if config.PROMETHEUS_ENABLED:
```

Comment:
Can you also add checks for the port/addr here (and below)?  If they are not set, then the `start_http_server` gives an error.

Reply:
done.